addons/sale/i18n/sale.pot

```diff
@@ -233,7 +233,7 @@ msgid ""
 "                <td>\n"
 "                    <strong>Shipping Method:</strong>\n"
 "                    ${object.carrier_id.name}\n"
-"                    % if object.carrier_id.fixed_price == 0.0:\n"
+"                    % if object.carrier_id.fixed_price == 0.0 or object.amount_delivery == 0.0:\n"
 "                        (Free)\n"
 "                    % else:\n"
 "                        (${format_amount(object.carrier_id.fixed_price, object.currency_id)})\n"
```

Comment:
Hello @Feyensv Thanks a bunch for having a check!

I made the fix based on a similar usage, which can be found when making the delivery fee part in the same XML file here: https://github.com/odoo/odoo/blob/13.0/addons/sale/data/mail_data.xml#L184-L191

For the first point you mentioned: `website_sale_delivery` is not installed, I did a few local tests to make sure the fix is valid. Before we generate the email, we first check if this is an order from the website here: https://github.com/odoo/odoo/blob/13.0/addons/sale/data/mail_data.xml#L142 If this is an order from the website, then the following email template parts will be taken into account. When we install the eCommerce and check the delivery cost in Sales module, the module `website_sale_delivery` is always installed. I also tested when we first install the Sales, then check the shipping costs, then install eCommerce, the module `website_sale_delivery` is still installed. This is assured here: https://github.com/odoo/odoo/blob/13.0/addons/website_sale/models/res_config_settings.py#L57-L73

If we create an order directly in Sales, e.g not from the website, the email template doesn't take into account the part under `if object.website_id` conditon, e.g: https://github.com/odoo/odoo/blob/13.0/addons/sale/data/mail_data.xml#L142-L283 So the email content for the orders created in Sales doesn't get affected.

Fixed price is not always correct because it's the predefined price for a fixed-price delivery method and it doesn't take into account the promotions or "free delivery when above an order amount". I kept the previous condition to avoid breaking something, do you think it's a better way to just use `amount_delivery` in the if condition?

For the translation issue, I kept the old translation `msgid` and the new one at the beginning, but that commit didn't pass the runbot test. So I replaced the it as it is now :crying_cat_face: 

If you prefer, I can make the fix to the master branch (I checked and it's still not fixed there). I think it's reasonable to do it as the template isn't get updated in the existing databases :smile: 

Could you let me know what do you think of my explanation and shall I retarget to master? Thanks a bunch! Cheers!